index.html

```diff
@@ -244,7 +244,7 @@ <h4 class="con-us">CONTACT US</h4>
                 </li>
                 <li class="li-item">
                   <label for="textarea">
-                    <textarea name="text" id="text"  class="text-input detail-input" placeholder="Enter text here..." cols="23" rows="5" max-length="500" required></textarea>
+                    <textarea class="text-input detail-input" name="text" id="text" cols="23" rows="5" placeholder="Enter text here..." max-length="500" required></textarea>
```

Comment:
- [ ] I kindly suggest that you correct your spelling of maxlength HTML attribute

:x: Wrong
`max-length="500"`

:heavy_check_mark: Right
`maxlength="500"`